.github/workflows/release.yaml

```diff
@@ -0,0 +1,83 @@
+on:
+  push:
+    branches:
+      - master
+name: release-please
+jobs:
+  # Create and maintain release PRs:
+  release-please-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - id: release-pr
+        uses: GoogleCloudPlatform/release-please-action@v2
+        with:
+          path: packages/gcf-utils
+          token: ${{ secrets.FORKING_TOKEN }}
+          fork: true
+          release-type: node
+          package-name: gcf-utils
+          command: release-pr
+      # Forking token does not have permission to add label, so we use
+      # this script to do so:
+      - id: label
+        if: ${{ steps.release-pr.outputs.pr }}
+        uses: actions/github-script@v3    
+        with:
+            github-token: ${{secrets.GITHUB_TOKEN}}
+            script: |
+              const [owner, repo] = process.env.GITHUB_REPOSITORY.split('/');
+              await github.issues.addLabels({
+                owner,
+                repo,
+                issue_number: ${{steps.release-pr.outputs.pr}},
+                labels: ['autorelease: pending']
+              });
+              console.log(`Tagged ${{steps.release-pr.outputs.pr}}`)
+  # Create a release and publish to npm, when release PRs are merged:
+  release-please-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GoogleCloudPlatform/release-please-action@v2
+        id: tag-release
+        with:
+          path: packages/gcf-utils
+          token: ${{ secrets.GITHUB_TOKEN }}
+          release-type: node
+          package-name: gcf-utils
+          command: github-release
+      - uses: actions/checkout@v2
+        if: ${{ steps.tag-release.outputs.release_created }}
+      - uses: actions/setup-node@v1
+        if: ${{ steps.tag-release.outputs.release_created }}
+        with:
+          node-version: 14
+          registry-url: 'https://wombat-dressing-room.appspot.com/'
+      # Actually publish to npm:
+      - name: publish
+        if: ${{ steps.tag-release.outputs.release_created }}
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
```

Comment:
Is this a token from wombat dressing room?  This would be the first place where we use a token like this in GitHub Secrets, right?

Reply:
This is a token from wombat dressing room. This is the same approach we're using for _googleapis_, to manage the monorepo, and for [release-please-action](https://github.com/google-github-actions/release-please-action).

Given that the token has 2FA tied to GitHub releases, and that wombat dressing room means we don't need to provide our actual OTP secrets in actions, I think this risk is worth the ability we gain of moving publication away from kokoro, which I've found limiting in its functionality compared to actions.